src/Metadata/Object/ViewObject.php

```diff
@@ -62,8 +67,9 @@ public function setIsUpdatable($isUpdatable)
         return $this;
     }
 
+    /** @return bool */
     public function isUpdatable()
     {
-        return $this->isUpdatable;
+        return (bool) $this->isUpdatable;
```

Comment:
Sneaky change here too, but I guess it's OK from a consumer PoV

Reply:
Consumer expects a boolean, so :shrug: